modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/TypeScriptNodeClientCodegen.java

```diff
@@ -128,4 +129,67 @@ public String getNpmRepository() {
     public void setNpmRepository(String npmRepository) {
         this.npmRepository = npmRepository;
     }
+
+    @Override
+    public Map<String, Object> postProcessAllModels(Map<String, Object> objs) {
+        Map<String, Object> result = super.postProcessAllModels(objs);
+
+        for (Map.Entry<String, Object> entry : result.entrySet()) {
```

Comment:
If you don't use entry.getKey in this loop, you can simplified this to:

`  for (Object o : result.values()) {
            if (o instanceof Map) {
              //some code
            }`